### PR TITLE
Add support for Swift Package Manager

### PR DESCRIPTION
**Pull request by kylef**

### CODE REVIEW

**Comment by kylef** [comment]
@Danappelxx  can you reivew/merge please :smile:.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: objective-c
 osx_image: xcode7.1
 script:
-- swiftc Nest.swift
+- swiftc sources/Nest.swift
```
Comment (kattrali) [inline .travis.yml]: Seems to be capitalised in some places but not here.

**Comment by kattrali** [comment]
:+1:

**Comment by Danappelxx** [comment]
Somebody beat me to it :smiley:

**Comment by kattrali** [comment]
Gotta be fast! :smile:

**Comment by iamDecode** [comment]
+1

**Pull request merged**